src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -217,6 +219,25 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
       }
     }
   }
+  /* Return a Markdown table of all entries in the database */
+  private def symbolsMarkdownTable(): Option[String] = {
+    val header = Seq("SCOPE","NAME","I/O","TYPE","VALUE")
+    val futureValue = fetchAllEntries.map {symbolStoreEntries =>
```

Comment:
The valuable part of this entire function is the individual unit and that's being mapped over. And that's the part which doesn't actually have a name.